content/en/news/releases/1.6.x/announcing-1.6.3/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Announcing Istio 1.6.3
+linktitle: 1.6.3
+subtitle: Patch Release
+description: Istio 1.6.3 patch release.
+publishdate: 2020-06-18
+release: 1.6.3
+aliases:
+    - /news/announcing-1.6.3
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.2 and Istio 1.6.3.
+
+{{< relnote >}}
+
+<!---
+Use proxyMetadata in gateway env (#24381) (#24590)
+--->
+## Changes
+- *Fixed* an issue preventing the operator from recreating watched resources if they are deleted ([#23238](https://github.com/istio/istio/issues/23238)).
```

Comment:
In the previous ones we did something like `(Issue 23238)` and not `(#23238)`. I don't mind if you don't keep it consistent with previous versions.

Reply:
I like the Issue ... format. Fixed. Thanks.